src/main/SubtitleListEditor.tsx

```diff
@@ -276,23 +276,30 @@ const SubtitleListSegment = React.memo((props: subtitleListSegmentProps) => {
   }
 
   // Maps functions to hotkeys
-  const handlers = {
-    addAbove: () => addCueAbove(),
-    addBelow: () => addCueBelow(),
-    jumpAbove: () => {
-      dispatch(setFocusSegmentTriggered(true))
-      dispatch(setFocusToSegmentAboveId({identifier: identifier, segmentId: cue.idInternal}))
-    },
-    jumpBelow: () => {
-      dispatch(setFocusSegmentTriggered(true))
-      dispatch(setFocusToSegmentBelowId({identifier: identifier, segmentId: cue.idInternal}))
-    },
-    delete: () => {
-      dispatch(setFocusSegmentTriggered(true))
-      dispatch(setFocusToSegmentAboveId({identifier: identifier, segmentId: cue.idInternal}))
-      deleteCue()
-    },
-  }
+  const hotkeyRef = useHotkeys<HTMLDivElement>([KEYMAP.subtitleList.addAbove.key, KEYMAP.subtitleList.addBelow.key, KEYMAP.subtitleList.jumpAbove.key, KEYMAP.subtitleList.jumpBelow.key, KEYMAP.subtitleList.delete.key],
+    (_, handler) => {
+      switch (handler.keys?.join('')) {
+        case KEYMAP.subtitleList.addAbove.key.split('+').pop():
+          addCueAbove()
+          break;
+        case KEYMAP.subtitleList.addBelow.key.split('+').pop():
+          addCueBelow()
+          break;
+        case KEYMAP.subtitleList.jumpAbove.key.split('+').pop():
+          dispatch(setFocusSegmentTriggered(true))
+          dispatch(setFocusToSegmentAboveId({identifier: identifier, segmentId: cue.idInternal}))
+          break;
+        case KEYMAP.subtitleList.jumpBelow.key.split('+').pop():
+          dispatch(setFocusSegmentTriggered(true))
+          dispatch(setFocusToSegmentBelowId({identifier: identifier, segmentId: cue.idInternal}))
+          break;
+        case KEYMAP.subtitleList.delete.key.split('+').pop():
+          dispatch(setFocusSegmentTriggered(true))
+          dispatch(setFocusToSegmentAboveId({identifier: identifier, segmentId: cue.idInternal}))
+          deleteCue()
+          break;
+      }
+    }, { enableOnFormTags: ['input', 'select', 'textarea'] }, [identifier, cue, props.index])
```

Comment:
Can't this be multiple `useHotkeys` calls? Here you are calling it once, with 5 different buttons and in the handler you `switch` over the actual key pressed. Unless I'm missing something having 5 `useHotkeys` calls is way easier, no?

Reply:
Probably. I don't know enough about `ref`s to properly assign multiple of them to a single `div`, so this was simply easier to write for me ^^'